ports/raspberrypi/boards/nullbits_bit_c_pro/mpconfigboard.mk

```diff
@@ -0,0 +1,9 @@
+USB_VID = 0x2E8A
+USB_PID = 0x6E61
```

Comment:
Hi, did you get the USB_PID from Raspberry Pi? I don't see it in the list here: https://github.com/raspberrypi/usb-pid/blob/main/Readme.md

Reply:
@dhalbert Gordon from Raspberry Pi confirmed USB_PID = 0x1048 and that the table will be updated soon. I've updated the PR with the correct PID.